pkg/templates/crds/hypershift/cluster.open-cluster-management.io_hypershiftdeployments.yaml

```diff
@@ -0,0 +1,7235 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Could you please use the latest version of crd? I think there are some updates. https://github.com/stolostron/hypershift-deployment-controller/blob/main/config/crd/cluster.open-cluster-management.io_hypershiftdeployments.yaml

Reply:
updated